src/main/java/org/apache/commons/validator/routines/UrlValidator.java

```diff
@@ -131,7 +131,7 @@ public class UrlValidator implements Serializable {
 
     private static final String AUTHORITY_REGEX =
             "(?:\\[(" + IPV6_REGEX + ")\\]|(?:(?:" + USERINFO_FIELD_REGEX + ")?([" + AUTHORITY_CHARS_REGEX + "]*)))(?::(\\d*))?(.*)?";
-    //             1                                 e.g. user:pass@           2                                       3       4
+    //             1                                 for example, user:pass@           2                                       3       4
```

Comment:
I think you need to remove some spaces here to make the numbers line up with the regex again